### ISSUE TITLE
Create UI/UX design 

### ISSUE THREAD

**Issue by Ank221199** [Ank221199/WorkoutGenerator#6]
## Expectation
Something mixture of these:
* https://www.muscleandstrength.com/workouts/10-week-mass-building-program.html  -- view the program 
* ![image](https://user-images.githubusercontent.com/40303711/193104042-56c6c7be-ac25-47aa-86e2-df7babfcbb58.png) -- while working out

**Comment by avezqureshi14** [comment]
I am thankful @Ank221199  to that you assigned me this , and I will give my best in delivering this to you

**Comment by Ank221199** [comment]
Dynamic API Functionality

**Comment by Ank221199** [comment]
This is a ticket for Design [Figma, adobe Xd]. After the design is completed we will go for development

**Comment by Ank221199** [comment]
> ed to be developed from scratch itself it will be better moving towards REACT JS this was my opinion, but what do you think?

Yeah, But my plan was to use VUE.js Since We want to continue the project.. and most of my friend circle use vue.js

**Comment by avezqureshi14** [comment]
OK @Ank221199

**Comment by sowmiya-venkatesan** [comment]
I'd love to contribute to this issue,   Please assign it to me @Ank221199

**Comment by avezqureshi14** [comment]
@Ank221199 my UI is ready should I make the Pull Request

**Comment by Ank221199** [comment]
Yeah, Plus if you can add screen shots that would be great to review

**Comment by avezqureshi14** [comment]
ok @Ank221199
